src/api/useDarkThemeMode.tsx

```diff
@@ -0,0 +1,11 @@
+import { useState } from "react";
```

Comment:
Why is this file in api folder? It has nothing to do with api I guess

Reply:
I moved it into seperate folder named "hooks" and it is inside src folder. Hope that is okay.